### ISSUE TITLE
[JetBrains, Java Renderer] Backslash ambiguous in oblique fonts

### ISSUE THREAD

**Issue by adamquaile** [chrissimpkins/Hack#67]
When a backslash appears in the italic/oblique font, it is understandably vertical. If it were still leaning back a little, it would help distinguish it from the pipe. 

Here is the backslash in oblique and regular. 
![backslash](https://cloud.githubusercontent.com/assets/69929/9606472/7e658882-50bb-11e5-8d6a-4f6fdc9b66e9.png)

**Comment by chrissimpkins** [comment]
Here is how it is designed to appear in all oblique: 
<img width="535" alt="backslash-char" src="https://cloud.githubusercontent.com/assets/4249591/9613132/967aca80-50b7-11e5-8cf1-fa9b80ecef9f.png">

Is this a JetBrains editor?  There are italics rendering issues with their Java renderer (Issue #26).  Based upon what I know to date, this seems to be limited to OS X versions only.

The Java renderer is taking the regular set and adding its own slant (too much) when these syntax highlighters use italics.  It doesn't occur with all fonts but it does affect others and I am trying to get feedback from JetBrains why this is an issue for Hack.

**Comment by adamquaile** [comment]
Yes, it is a JetBrains editor, on Mac too, so it sounds like this is a known bug then. Thanks.

**Comment by chrissimpkins** [comment]
If you intend to continue to use the font, keep an eye on that thread and I will add any additional information that I receive.  I am not terribly hopeful that this is going to land on their to-do list soon because there are reports of this problem that date back over a year in their issue tracker.

**Comment by adamquaile** [comment]
I've only been using it from today, but this is the only issue I've noticed and for the kind of environment I'm in it's a pretty minor one. 

It doesn't appear often and I'll just turn italics off where it does. Nice work, by the way - I'm liking it so far.

**Comment by chrissimpkins** [comment]
Really glad to hear that Adam.  Let me know if you come across any other issues.


—
Chris

**Comment by chrissimpkins** [comment]
Will you please give these new builds a try (v2.012)?  I've updated the name tables and adjusted the font names based upon what is currently recommended for cross-platform support.  Let's see if this addresses this issue.

**Download Links**
[Regular](https://github.com/chrissimpkins/Hack/blob/development/build/test_builds/2_012/Hack-Regular.ttf?raw=true)
[Bold](https://github.com/chrissimpkins/Hack/blob/development/build/test_builds/2_012/Hack-Bold.ttf?raw=true)
[Italic](https://github.com/chrissimpkins/Hack/blob/development/build/test_builds/2_012/Hack-Italic.ttf?raw=true)
[BoldItalic](https://github.com/chrissimpkins/Hack/blob/development/build/test_builds/2_012/Hack-BoldItalic.ttf?raw=true)

**Comment by adamquaile** [comment]
I've just tried this on the same platform (not the same computer) and the backslash looks perfect.

**Comment by chrissimpkins** [comment]
Great!  Progress... (and potentially a few closed issue reports).  Will merge this into master this weekend.  Thank you very much for the report. It happened to be associated with a number of other reported issues.  I will close this once it is merged.

**Comment by chrissimpkins** [comment]
Thank you very much for reporting this issue Adam.  The fix is now included in the v2.013 release which was merged into the master branch.  I will close this issue as fixed.
